packages/analytics/analytics-chart/src/components/chart-types/TimeSeriesChart.vue

```diff
@@ -228,6 +246,26 @@ watch(() => props.type, () => {
   delete verticalLinePlugin.clickedSegment
 })
 
+
+nextTick(() => {
+  if (chartInstance.value?.chart) {
```

Comment:
Isn't `nextTick` firing every time Vue applies updates?  That seems like it could potentially apply a ton of redundant event listeners, unless I'm missing something.

As long as it's only applying the listeners once, I don't think I have an opinion; we just want to make sure the `zoom` prop is reactive if at all possible.

Reply:
because the reactivity of the zoom prop requires remounting of the underlying chart components in order to update plugins, event listeners need to be removed/re-added whenever the chart instance updates. Updated to use `watch` here.

Reply:
That makes sense; I think it's fine.